input/ig.xml

```diff
@@ -396,135 +396,135 @@
 		<resource>
 			<reference>
 				<reference value="Observation/pulserate-1"/>
 			</reference>
 			<name value="Observation - Pulse rate"/>
-			<description value="Pulse rate recorded by GP&#xa;&#xa;Patient: Mr. Li Wang"/>
+			<description value="Shows an example of a pulse rate observation for the *AU Core Heart Rate* profile. Patient: Mr. Li Wang."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-heartrate"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/lipid-hdl-1"/>
 			</reference>
 			<name value="Observation - HDL"/>
-			<description value="HDL result&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shows an example of an HDL result for the *AU Core Pathology Result Observation* profile. Patient: Mia Leanne Banks."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/lipid-ldl-1"/>
 			</reference>
 			<name value="Observation - LDL"/>
-			<description value="LDL result&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shows an example of an LDL result for the *AU Core Pathology Result Observation* profile. Patient: Mia Leanne Banks."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/lipid-total-chol-1"/>
 			</reference>
 			<name value="Observation - Total Cholesterol/HDL ratio"/>
-			<description value="Total Cholesterol/HDL ratio result&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shows an example of a Total Cholesterol/HDL ratio result for the *AU Core Pathology Result Observation* profile. Patient: Mia Leanne Banks."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/lipid-triglyceride-1"/>
 			</reference>
 			<name value="Observation - Triglyceride"/>
-			<description value="Triglyceride result&#xa;&#xa;Patient: Ms. Mia LEANNE BANKS"/>
+			<description value="Shows an example of a Triglyceride result for the *AU Core Pathology Result Observation* profile. Patient: Mia Leanne Banks."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/pathresult-covid-1"/>
 			</reference>
 			<name value="Observation - COVID Pathology Result (Not detected)"/>
-			<description value="COVID Pathology Result (Not detected)&#xa;&#xa;Patient: Mr. Li Wang"/>
+			<description value="Shows an example of a COVID-19 PCR test result indicating 'Not detected' for the *AU Core Diagnostic Result - Pathology* profile, with notes on test accreditation and evaluation status. Patient: Li Wang."/>
```

Comment:
reorder the words so the profile is at the end. Should say:

> Shows an example of a COVID-19 PCR test result indicating 'Not detected' with notes on test accreditation and evaluation status for the *AU Core Diagnostic Result - Pathology* profile. Patient: Li Wang.

Reply:
Done